Makefile

```diff
@@ -62,6 +62,7 @@ $(DIST):
 	mkdir -p $(DISTDIR)
 	npm install || (cat npm-debug.log && exit 1)
 	npm run build
+	npm install --production || (cat npm-debug.log && exit 1)
```

Comment:
I'm not sure why, but it seems it doesn't work here - meaning it doesn't remove dev dependencies.

I checked it out in MU and it seems to work there (calling `npm install` followed by `npm install --production` removes unnecessary dependencies). But in here it seems to do nothing.